.github/workflows/ci.yml

```diff
@@ -319,35 +280,37 @@ jobs:
         PRINT_GLXINFO: true
       run: ./tests/integration/run_tests_headless.sh
 
-  test-parse:
+
+  test_parse:
+    needs: [changed, build_windows]
     name: Data Files
-    needs: [build_windows, changed]
-    if: ${{ needs.changed.outputs.game_code == 'true' || needs.changed.outputs.data == 'true' || needs.changed.outputs.integration_tests == 'true' }}
-    runs-on: ${{ matrix.os }}
+    if: ${{ (needs.changed.outputs.game_code == 'true' || needs.changed.outputs.cmake == 'true' || needs.changed.outputs.data == 'true' || needs.changed.outputs.parse_script == 'true') && always() }}
+    runs-on: windows-2022
     env:
-      DIR_MINGW64: C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64
-    strategy:
-      matrix:
-        os: [windows-2022]
+      CONTINUOUS: EndlessSky-win64-continuous.zip
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     steps:
     - uses: actions/checkout@v3
-    - name: Install runtime dependencies
+    # If no code changes occurred then we can download the latest continuous.
+    - name: Download latest continuous
+      if: ${{ needs.changed.outputs.game_code != 'true' && needs.changed.outputs.cmake != 'true' }}
```

Comment:
why not `== 'false'`?

Reply:
That doesn't actually work funnily enough. Apparently it should but when I tried it the step was skipped anyways.